diff -Nur File-Rsync-0.42.ori/Makefile.PL File-Rsync-0.42.mod/Makefile.PL
--- File-Rsync-0.42.ori/Makefile.PL	2005-06-25 00:02:32.000000000 +0200
+++ File-Rsync-0.42.mod/Makefile.PL	2006-01-22 13:43:33.000000000 +0100
@@ -17,10 +17,10 @@
       AUTHOR         => $AUTHOR,
       ABSTRACT       => $ABSTRACT,
       PREREQ_PM      => { 'Scalar::Util' => 0 },
-      PL_FILES       => { 'makepm.PL' => 'config.pm' },
+      PL_FILES       => { 'makepm.PL' => 'Config.pm' },
       PM             => { 'Rsync.pm'  => '$(INST_LIB)/File/Rsync.pm',
-     			  'config.pm' => '$(INST_ARCHLIB)/File/Rsync/Config.pm' },
+     			  'Config.pm' => '$(INST_LIB)/File/Rsync/Config.pm' },
       dist           => { COMPRESS => 'gzip', SUFFIX => 'gz' },
-      realclean      => { FILES => 'config.pm' },
+      realclean      => { FILES => 'Config.pm' },
 );
 
diff -Nur File-Rsync-0.42.ori/makepm.PL File-Rsync-0.42.mod/makepm.PL
--- File-Rsync-0.42.ori/makepm.PL	2005-06-24 23:56:33.000000000 +0200
+++ File-Rsync-0.42.mod/makepm.PL	2006-01-22 13:42:09.000000000 +0100
@@ -20,15 +20,15 @@
 (this is not recommended)).
 
 EOT
-$RSYNC||='/usr/local/bin/rsync';
-print "Path to rsync [$RSYNC]: ";
-my $ans=<STDIN>;
-chomp $ans;
-$RSYNC=$ans || $RSYNC;
-$RSYNC=~s/\\/\\\\/g;
-$RSYNC=~s/\'/\\\'/g;
+$RSYNC||='/usr/bin/rsync';
+#print "Path to rsync [$RSYNC]: ";
+#my $ans=<STDIN>;
+#chomp $ans;
+#$RSYNC=$ans || $RSYNC;
+#$RSYNC=~s/\\/\\\\/g;
+#$RSYNC=~s/\'/\\\'/g;
 open IN,'config.in' or die "Cannot read config.in: $!\n";
-open OUT,'>config.pm' or die "Cannot write config.pm: $!\n";
+open OUT,'>Config.pm' or die "Cannot write Config.pm: $!\n";
 while (<IN>) {
    if (/^(\s*rsync_path\s*=>)/) { print OUT "$1 '$RSYNC',\n" }
    else                                  { print OUT }
